### PR TITLE
Update text to say we support "all SGX/Enclave supported s/w" platforms, instead of "all s/w platforms"

### LINKED ISSUE #2610: [WEBSITE] Add "macOS" or remove "all software platforms"
Visiting the website

https://openenclave.io/sdk/

the section "Core Tenents" mentions:

> Multi-platform
> Design with all software platforms, Windows and Linux, in mind

This needs to be corrected, either:
* Add macOS support
* Correct "all software platforms"

I usually write within projects "you can't ignore one of the major 3 platforms, which is Windows".

Well, now I write "you can't ignore one of the major 3 platforms, which is macOS"

### PR DESCRIPTION
**Pull request by radhikaj**
Fix https://github.com/openenclave/openenclave/issues/2610.

### CODE REVIEW

File: sdk/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ An enclave application partitions itself into two components (1) an untrusted co
    <div class="column3"  ><img src="../assets/images/Standardized.png" width="50" /><h4>Standardized</h4><p>Remove hardware vendor specific signing and verification requirements</p></div>
   </div>
 <div class="row3">
-   <div class="column3"><img src="../assets/images/Multiplatform.png" width="50" /><h4>Multi-platform</h4><p>Design with all software platforms, Windows and Linux, in mind</p></div>
+   <div class="column3"><img src="../assets/images/Multiplatform.png" width="50" /><h4>Multi-platform</h4><p>Design with all SGX/enclave supported software platforms, Windows and Linux, in mind</p></div>
```
Comment (jhand2) [inline sdk/index.md]: I don't really think this is what we want. The goal is to support any platform with TEE support. How about

> "Design with all TEE supported software platforms in mind"

Today that's Windows and Linux, tomorrow who knows. This is just saying what the protect goals are, we list what OS's we officially support elsewhere.

**Review by jhand2 — APPROVED** [review]
Perfect 😄

**Comment by lazaridiscom** [comment]
Latest change:

> "Multi-platform</h4><p>Design with Windows and Linux in mind</p></div>

This ignores again macOS. To not ignore macOS without mentioning it, use something like:

=> Design with all TEE supported software platforms  in mind, (currently Windows and Linux)

**Comment by jhand2** [comment]
> 
> 
> Latest change:
> 
> > "Multi-platformDesign with Windows and Linux in mind
> 
> This ignores again macOS. To not ignore macOS without mentioning it, use something like:
> 
> => Design with all TEE supported software platforms in mind, (currently Windows and Linux)

@lazaridiscom I disagree. We support Windows and Linux and we state that. If we supported macOS we would say so. But we don't.

**Comment by jhand2** [comment]
@radhikaj can you squash your commits and add DCO sign-off?

**Comment by lazaridiscom** [comment]
@radhikaj  started with

> Design with all SGX/enclave supported software platforms, Windows and Linux, in mind

which was near perfect.

you asked for:

>How about
>    "Design with all TEE supported software platforms in mind"

which was incomplete (does not mention the platforms)

@radhikaj change to

> Design with Windows and Linux in mind

which removed again the "TEE" / "SGX/Enclave" clarification, so we are where we started:

This is not about agreeing. "Multi-Platform", today, in 2020, means at minimum "Win, Lin, Mac".

And you ignore "Mac" which (to be blunt) is amateurish.

**Pull request closed without merging**